### ISSUE TITLE
Feature request - CDN hosting

### ISSUE THREAD

**Issue by digimbyte** [rentfuse-labs/neo-wallet-adapter#31]
Looking for a simple import of the scripts to be used as a library, no pointless react components, just raw API without an NPM project like react, ant, or angular

**Comment by mfbz** [comment]
Hi, you could just:

`npm install @rentfuse-labs/neo-wallet-adapter-wallets @rentfuse-labs/neo-wallet-adapter-base`

and use basic functionalities in our dApp without react stuff.
Is this solving your issue?

**Comment by digimbyte** [comment]
No, because its not a node project. its raw html and dom manipulation using min-vue

**Comment by mfbz** [comment]
Gotcha 👍

**Comment by mfbz** [comment]
@vncoelho nothing at the moment, we planned this for the end of August / September

**Comment by mfbz** [comment]
Closed in #34! 🚀 

To use it you can do like this:

```html
<!DOCTYPE html>
<html>
  ...
  <script src="https://www.unpkg.com/@rentfuse-labs/neo-wallet-adapter-bundle@0.3.0/lib/neo-wallet-adapter.web.js"></script>
  <script>
    // ...
    // Global variable
    NeoWalletAdapter.fun1('Five');
    // Property in the window object
    window.NeoWalletAdapter.fun1('Five');
    // ...
  </script>
</html>
```
